src/common/utils/document.ts

```
@@ -1,6 +1,6 @@
 import { WrappedDocument, getData, v2 } from "@govtechsg/open-attestation";
 
-// TODO: change this to use the open-attestation library exported utilities when we upgrade the version
-export const getDocumentId = (document: WrappedDocument): string => `0x${document.signature.targetHash}`;
+export const getDocumentId = (document: WrappedDocument<v2.OpenAttestationDocument>): string =>
```

three things

extract getDocumentId to OpenAttestation library, follow the style of getMerkleRoot -> name it getTargetHash and appropriately handle v2 and v3 differences
https://github.com/Open-Attestation/open-attestation/blob/3b1c1d5e74231e4404c6088aadf107e157d0634c/src/shared/utils/utils.ts#L82-L82

create another function in open-attestation that's called getAssetId and it should throw an error if the document isn't a transferable asset - then import that function wherever this one is used

replace all usages of the function below with open-attestation/utils/getIssuerAddress

---

ok, waiting for review of https://github.com/Open-Attestation/open-attestation/pull/174 as this will need the new functions in OA utils.
- [x] update getTargetHash
- [x] update getAssetId
- [x] update getIssuerAddress